content/spin/api-guides-overview.md

```diff
@@ -0,0 +1,24 @@
+title = "API Guides Overview"
+template = "spin_main"
+date = "2023-03-03T03:03:03Z"
+
+---
+
+### Spin's Hosting Capabilities and Interfaces
+
+Spin is a multi-language framework for [writing server-like applications](https://developer.fermyon.com/spin/writing-apps) that compile into WebAssembly (Wasm). The Spin framework provides capabilities for your Wasm-powered application by offering a set of interfaces. For example, making sufficient HTTP functionality available to [send outbound HTTP requests](https://developer.fermyon.com/spin/http-outbound) and more.
```

Comment:
I would be inclined to lose the "for example" - the page is about to provide a comprehensive list of examples!  _grin_

Actually... I would be inclined to lose this whole paragraph (and have a super simple 'get out of the way and let me at the table' intro like the language support overview).  The exception would be for a brief "why do I need all this when my beloved language already has libraries for this stuff which I know inside and out and don't want to have to learn some random alternative" explained, though the pages on "client APIs" already have that so certainly not a must-have.

Reply:
Done